charts/tidb-lightning/templates/job.yaml

```diff
@@ -37,7 +40,7 @@ spec:
           mountPath: /etc/rclone
         - name: data
           mountPath: /data
-      {{- if and (ne .Values.timezone "UTC") (ne .Values.timezone "") }}
+      {{ if and (ne .Values.timezone "UTC") (ne .Values.timezone "") }}
```

Comment:
why no dash?

Reply:
I was seeing the following:
```volumeMounts:
        - name: credentials
          mountPath: /etc/rclone
        - name: data
          mountPath: /datarestartPolicy: Never      # if lightning fails, manual intervention is required so no restart```

Reply:
Does the latest tidb-lightning exit with 0 when loading successfully now?

Reply:
@tennix that is what I saw, yes.

Reply:
Then I think we can remove the fail-fast hack script.

Reply:
@tennix I removed the script as well as the fail fast option